lib/matchers/matchers.rb

```diff
@@ -222,6 +222,44 @@
   end
 end
 
+# `be_in` matcher
+# You can use it in the following cases:
+# - check if an item or array is included in a given array
+# eg:
+# describe nginx do
+#   its('user') { should be_in AUTHORIZED_USER_LIST }
+# end
+# describe nginx do
+#   its('module_list') { should be_in AUTHORIZED_MODULE_LIST }
+# end
+RSpec::Matchers.define :be_in do |list|
+  match do |item|
+    # Handle both single item and array
+    item.is_a?(Array) ? (item-list).empty? : list.include?(item)
```

Comment:
Normal Ruby styling includes a space around operators. Can you please make this `(item - list)` so it's clearer?